po/pl.po

```diff
@@ -0,0 +1,17886 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: Comprehensive Rust 🦀\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: 2023-03-10 17:32+0100\n"
+"Last-Translator: Kuba Jaroszewski <jakub.jaroszewski@gmail.com>\n"
+"Language-Team: Polish <translation-team-pl@lists.sourceforge.net>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: pl\n"
+"Plural-Forms: nplurals=3; plural=(n==1 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 "
+"|| n%100>=20) ? 1 : 2);\n"
+
+#: src/SUMMARY.md:3
+msgid "Welcome to Comprehensive Rust 🦀"
+msgstr "Witamy w Comprehensive Rust 🦀"
+
+#: src/SUMMARY.md:4
+msgid "Running the Course"
+msgstr "Prowadzenie kursu"
+
+#: src/SUMMARY.md:5
+msgid "Course Structure"
+msgstr "Struktura kursu"
+
+#: src/SUMMARY.md:6
+msgid "Keyboard Shortcuts"
+msgstr "Skróty klawiszowe"
+
+#: src/SUMMARY.md:7
+msgid "Translations"
+msgstr "Tłumaczenia"
+
+#: src/SUMMARY.md:8
+msgid "Using Cargo"
+msgstr "Korzystanie z Cargo"
+
+#: src/SUMMARY.md:9
+msgid "Rust Ecosystem"
+msgstr "Ekosystem Rust"
+
+#: src/SUMMARY.md:10
+msgid "Code Samples"
+msgstr "Próbki kodu"
+
+#: src/SUMMARY.md:11
+msgid "Running Cargo Locally"
+msgstr "Lokalne uruchamianie Cargo"
+
+#: src/SUMMARY.md:14
+msgid "Day 1: Morning"
+msgstr "Dzień 1: Rano"
+
+#: src/SUMMARY.md:18 src/SUMMARY.md:74 src/SUMMARY.md:127 src/SUMMARY.md:183
+msgid "Welcome"
+msgstr "Witamy"
+
+#: src/SUMMARY.md:19
+msgid "What is Rust?"
+msgstr "Co to jest Rust?"
+
+#: src/SUMMARY.md:20
+msgid "Hello World!"
+msgstr "Witaj świecie!"
+
+#: src/SUMMARY.md:21
+msgid "Small Example"
+msgstr "Mały przykład"
+
+#: src/SUMMARY.md:22
+msgid "Why Rust?"
+msgstr "Dlaczego Rust?"
+
+#: src/SUMMARY.md:23
+msgid "Compile Time Guarantees"
+msgstr "Gwarancje w czasie kompilacji"
+
+#: src/SUMMARY.md:24
+msgid "Runtime Guarantees"
+msgstr "Gwarancje podczas działania"
+
+#: src/SUMMARY.md:25
+msgid "Modern Features"
+msgstr "Nowoczesne funkcje"
```

Comment:
Hi @pawelpaa, awesome that you're reviewing this!

Just a small tip about GitHub suggestions (something which I wish I would have known much earlier): you can use the left-most button above to do this:

```suggestion
msgstr "Nowoczesne funkcjonalności"
```

:exploding_head:  :smile: 

This way, the suggestion can be applied with a single click afterwards (you can even apply it yourself most of the time). That has saved me a ton of time because of fewer round-trips on the PRs here.